spec/models/article_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  it "creates articles" do
+    article_data = [{
+      title: "title",
+      url: "url",
+      abstract: "abstract",
+      desc_facet: "desc_facet",
+      geo_facet: "geo_facet"
+    }]
+
+    Article.create_articles(article_data)
+
+    expect(Article.first.title).to eq('title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.